packages/@aws-cdk/aws-events-targets/lib/apigateway.ts

```diff
@@ -0,0 +1,77 @@
+import * as apigateway from '@aws-cdk/aws-apigateway';
+import * as events from '@aws-cdk/aws-events';
+import * as iam from '@aws-cdk/aws-iam';
+import { singletonEventRole } from './util';
+
+/**
+ * Customize the API Gateway Event Target
+ */
+export interface ApiGatewayProps {
+  /**
+   * Method name
+   *
+   * @default `*`
+   */
+  readonly method?: string;
+
+  /**
+   * Resource path
+   *
+   * @default `*`
+   */
+  readonly path?: string;
+
+  /**
+   * Stage name
+   *
+   * @default `*`
+   */
+  readonly stage?: string;
```

Comment:
is any of this information redundant with `IRestApi`? - at first glance it looks like stage might be?